Source/Scene/PointCloudEyeDomeLighting.js

```diff
@@ -0,0 +1,400 @@
+        this._drawCommands = undefined;
+        this._clearCommands = undefined;
+
+        this._strength = 1.0;
+        this._radius = 1.0;
+    }
+
+    function addConstants(sourceFS, constantName, value) {
+        var finalSource = sourceFS;
+        if (typeof(value) === 'boolean') {
+            if (value !== false) {
+                finalSource = '#define ' + constantName + '\n' + sourceFS;
+            }
+        } else {
+            finalSource = '#define ' + constantName + ' ' + value + '\n' + sourceFS;
+        }
+        return finalSource;
+    }
+
+    function createSampler() {
+        return new Sampler({
+            wrapS : TextureWrap.CLAMP_TO_EDGE,
+            wrapT : TextureWrap.CLAMP_TO_EDGE,
+            minificationFilter : TextureMinificationFilter.NEAREST,
+            magnificationFilter : TextureMagnificationFilter.NEAREST
+        });
+    }
+
+    function destroyFramebuffers(processor) {
+        var framebuffers = processor._framebuffers;
+        if (!defined(framebuffers)) {
+            return;
+        }
+
+        processor._colorTexture.destroy();
+        processor._ecAndLogDepthTexture.destroy();
+        processor._depthTexture.destroy();
+        for (var name in framebuffers) {
+            if (framebuffers.hasOwnProperty(name)) {
+                framebuffers[name].destroy();
+            }
+        }
+
+        processor._framebuffers = undefined;
+        processor._colorTexture = undefined;
+        processor._ecAndLogDepthTexture = undefined;
+        processor._depthTexture = undefined;
+        processor._drawCommands = undefined;
+        processor._clearCommands = undefined;
+    }
+
+    function createFramebuffers(processor, context) {
+        var screenWidth = context.drawingBufferWidth;
+        var screenHeight = context.drawingBufferHeight;
+
+        var colorTexture = new Texture({
+            context : context,
+            width : screenWidth,
+            height : screenHeight,
+            pixelFormat : PixelFormat.RGBA,
+            // Firefox as of 57.02 throws FRAMEBUFFER_UNSUPPORTED 0x8CDD if this doesn't match what's in ecTexture
+            pixelDatatype : FeatureDetection.isFirefox() ? PixelDatatype.FLOAT : PixelDatatype.UNSIGNED_BYTE,
+            sampler : createSampler()
+        });
+
+        var ecTexture = new Texture({
+            context : context,
+            width : screenWidth,
+            height : screenHeight,
+            pixelFormat : PixelFormat.RGBA,
+            pixelDatatype : PixelDatatype.FLOAT,
+            sampler : createSampler()
+        });
+
+        var depthTexture = new Texture({
+            context : context,
+            width : screenWidth,
+            height : screenHeight,
+            pixelFormat : PixelFormat.DEPTH_COMPONENT,
+            pixelDatatype : PixelDatatype.UNSIGNED_INT,
+            sampler : createSampler()
+        });
+
+        processor._framebuffers = {
+            prior : new Framebuffer({
```

Comment:
Since there is only one framebuffer just make the single framebuffer it's own variable and remove `_framebuffers`.

Reply:
Similarly, `_drawCommands` only has `blendCommand` and `clearCommands` only has `prior`.